packages/node-http-handler/src/write-request-body.ts

```diff
@@ -59,7 +59,14 @@ function writeBody(
     // pipe automatically handles end
     body.pipe(httpRequest);
   } else if (body) {
-    httpRequest.end(Buffer.from(body as Parameters<typeof Buffer.from>[0]));
+    if (Buffer.isBuffer(body) || typeof body === "string") {
+      httpRequest.end(body);
```

Comment:
ClientRequest (`node:http(s)`) and ClientHttp2Stream (`node:http2`) both accept `.end(string)` in addition to `.end(Buffer)`.